lib/src/commands/delete/delete_view_commad.dart

```diff
@@ -41,13 +41,22 @@ class DeleteViewCommand extends Command with ProjectStructureValidator {
       help: 'The length of the line that is used for formatting',
       valueHelp: '80',
     );
+
+    argParser.addOption(
```

Comment:
@ferrarafer does adding this here mean we have to pass the config path every time we run a command?